dandiapi/api/migrations/0006_default_oauth_application.py

```diff
@@ -0,0 +1,42 @@
+# flake8: noqa N806
+
+from django.db import migrations
+from django.db.models import Q
+
+
+def create_application(apps, schema_editor):
+    Application = apps.get_model('oauth2_provider', 'Application')
+    name = 'DANDI GUI'
+    # This is specified so it matches the default value used in the frontend.
+    client_id = 'Dk0zosgt1GAAKfN8LT4STJmLJXwMDPbYWYzfNtAl'
+    if not Application.objects.filter(Q(name=name) | Q(client_id=client_id)).exists():
+        application = Application(
+            # Production instances should change this.
+            client_id='Dk0zosgt1GAAKfN8LT4STJmLJXwMDPbYWYzfNtAl',
+            # Production instances must change this.
+            redirect_uris='http://localhost:8085/',
```

Comment:
set env var to provide it?

Reply:
`http://localhost:8085/` is always correct for this context.